src/test/java/io/lakefs/iceberg/TestCatalogMigration.java

```diff
@@ -107,12 +108,13 @@ public void testMigrateHadoopToLakeFSCatalog() throws NoSuchTableException {
         sharedSpark.sql(String.format("CREATE TABLE IF NOT EXISTS %s USING iceberg AS SELECT * FROM %s", lakeFSFullTableName,hiveFullTableName));
 
         // show cloned table in lakeFS
-        var lakeFSDf = sharedSpark.sql(String.format("SELECT * FROM %s.%s.%s.%s", lakeFSCatalog, branch, db, table));
+        Dataset<Row> lakeFSDf;
+        lakeFSDf = sharedSpark.sql(String.format("SELECT * FROM %s.%s.%s.%s", lakeFSCatalog, branch, db, table));
         lakeFSDf.show();
 
         // assert source and target tables are equal
-        var diff = lakeFSDf.except(sharedSpark.sql(String.format("SELECT * FROM %s", hiveFullTableName)));
-        assertEquals(true, diff.isEmpty());
+        Dataset<Row> diff = lakeFSDf.except(sharedSpark.sql(String.format("SELECT * FROM %s", hiveFullTableName)));
+        assertTrue(diff.isEmpty());
```

Comment:
If this does not print a useful error message, then it will be hard to understand failure messages here.  It may be easier to understand them if we used some JUnit helper that tests if the Container `diff.collect()` is empty.